Cargo.toml

```diff
@@ -38,12 +38,13 @@ categories = ["cryptography"]
 zksync_mini_merkle_tree = { git = "https://github.com/matter-labs/zksync-era", rev = "core-v27.0.0" }
 zksync_multivm = { git = "https://github.com/matter-labs/zksync-era", rev = "core-v27.0.0" }
 zksync_contracts = { git = "https://github.com/matter-labs/zksync-era", rev = "core-v27.0.0" }
+zksync_basic_types = { git = "https://github.com/matter-labs/zksync-era", rev = "core-v27.0.0" }
```

Comment:
`zksync_types` should include everything from `zksync_basic_types`. Any reason you added this dependency?

Reply:
`zksync_error` requires the basic types crate. There is good chance that we'll use `zksync_error` inside `zksync_types` for their public errors, so `zksync_error` can't be dependent on `zksync_types` otherwise we will end up with circular dependency.

Reply:
But `zksync_error` (anvil-zksyn's crate) does not depend on `zksync_basic_types` right now? Unless I am missing something

Reply:
it is, if i unserstood you correctly:

https://github.com/matter-labs/anvil-zksync/blob/0df86310c95519c02b755c58d62ac1fd145d6f6b/crates/zksync_error/Cargo.toml#L14

Reply:
Ah, indeed. Sorry, I looked at the wrong place